### PR TITLE
feat: remove iwdp related capabilities, objects

### PR DESCRIPTION
**Pull request by KazuCocoa**
Does https://github.com/appium/appium-xcuitest-driver/pull/1020 means this?
(Get rid of IWDP related caps and instances)

Or leave this for a while to be able to back to IWDP way

### CODE REVIEW

**Comment by umutuzgur** [comment]
You are right. I think we just forgot to remove these

File: README.md [hunk 1]
```diff
@@ -164,8 +164,6 @@ Differences are noted here:
...
 |`maxTypingFrequency`|Maximum frequency of keystrokes for typing and clear. If your tests are failing because of typing errors, you may want to adjust this. Defaults to 60 keystrokes per minute.|e.g., `30`|
 |`simpleIsVisibleCheck`|Use native methods for determining visibility of elements. In some cases this takes a long time. Setting this capability to `false` will cause the system to use the position and size of elements to make sure they are visible on the screen. This can, however, lead to false results in some situations. Defaults to `false`, except iOS 9.3, where it defaults to `true`.|e.g., `true`, `false`|
-|`startIWDP`|Set this to `true` if you want to start ios_webkit_debug proxy server automatically for accessing webviews on iOS. The capatibility only works for real device automation. Defaults to `false`.|e.g., `true`|
-|`webkitDebugProxyPort`|Local port number used for communication with ios-webkit-debug-proxy. Only relevant for real devices. The default value equals to `27753`.|e.g. `20000`|
```
Comment (mykola-mokhnach) [inline README.md]: Don'r we need this setting anymore? Are the ports assigned automatically when parallel testing sessions connect to web views @umutuzgur ?

Comment (umutuzgur) [reply README.md]: @mykola-mokhnach There are not server ports anymore. `appium-remote-debugger` talks to the phone directly over `usbmuxd`

Comment (mykola-mokhnach) [reply README.md]: good, then the cap should be removed from desired caps and other places where it is used as well

Comment (KazuCocoa) [reply README.md]: Yea, we should find them in appium doc like https://github.com/appium/appium/blob/4fb04006157e56656324863f22f0d6501aad08f8/docs/en/advanced-concepts/parallel-tests.md and appium-ios-driver.
(in this repo, webkitDebugProxyPort was just passed to ios-driver'iwdp)

**Comment by imurchie** [comment]
Thanks for this! I completely forgot about this stuff.

The `appium-ios-driver` will continue to use the old system, so there is no need to remove it there. and the main documentation needs to change to reflect that this driver, going forward, works differently.

**Pull request merged**